### ISSUE TITLE
Missing search bar?

### ISSUE THREAD

**Issue by amcooper** [wallabag/ios-app#271]
| Q             | A
| ------------- | ---
| Device?       | iPhone
| Device Os version    | 13.4
| App Version   | 5.0.1 build 257
| Wallabag server version | 2.0

Just downloaded the app on 2020-04-15 and have not been able to make the search bar appear. See screenshot:
![](https://i.imgur.com/9Yjpb47.jpg)

### Steps to reproduce/test case
N/A

**Comment by bourvill** [comment]
Yes, not search bar actually, is a missing feature from last migration like text to speech.

**Comment by amcooper** [comment]
Oh, I see the update is on its way. I eagerly wait!
